centralized-sampling-tests/README.md

```diff
@@ -0,0 +1,38 @@
+# Centralized Sampling Integration Tests
+
+## Introduction
+
+This project is meant to be used to test that X-ray's centralized sampling strategies
+are working properly. This folder has sample apps configured for the integration tests
+under sample apps in languages that support centralized sampling. As of now the languages
+supported are Java and Go. To run these tests, first start up the collector, then
+start up the chosen sample app, then start running the tests.
+
+## Run Locally
+
+### Set up collector
+To run locally first set up the collector with the correct configuration.
+It is possible that this is already done. The collector-contrib releases can be found here
+[aws-otel-collector](https://github.com/aws-observability/aws-otel-collector).
+Make sure that the collector config is configured to work with a local x-ray listener pointed
+to port 2000. See the example-collector-config.yaml for what it should look like. Start the 
+collector with command.
+```shell
+    docker run --rm -p 2000:2000 -p 55680:55680 -p 8889:8888 \
+      -e AWS_REGION=us-west-2 \
+      -e AWS_PROFILE=default \
+      -v ~/.aws:/root/.aws \
+      -v "${PWD}/examples/docker/config-test.yaml":/otel-local-config.yaml \
+      --name awscollector public.ecr.aws/aws-observability/aws-otel-collector:latest \
+      --config otel-local-config.yaml;
+```
+
+### Start up sample app
+Start up the sample app of your choice that is configured for centralized sampling integration
+tests. The sample apps exist in the sample-apps folder. Each sample app will have a 
```

Comment:
Which sample-apps in the sample app folder are configured for centralized sampling?

Reply:
realistically all of the sample apps in this subfolder would be, but that comment was for if other sample apps get added later on when I'm gone for some reason that aren't configured for the testing. Might not be necessary to include that